modules/sievefilters/modules.php

```diff
@@ -1278,6 +1283,16 @@ function sieve_enabled_callback($val) { return $val; }
     }
 }
 
+/**
```

Comment:
@Shadow243 I think the goal is to have both redirect and forward here. I think there is no need to create a user setting for this.

![Screenshot 2024-07-31 at 10 13 58](https://github.com/user-attachments/assets/f34c1d10-3f9d-40cc-9a2e-5185a7607742)